app/scripts/lib/experiments/base.js

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// An experiment interface.
```

Comment:
Can you expand the description to talk about what this module is supposed to do? It'll make it a bit easier to review too.